infrastructure/conf/nginx-conf-local/default

```diff
@@ -85,6 +85,8 @@ server {
 
     # rewrite root-level possible lang prefix to talentsearch index.html
     location ~ "^(?:/[a-z]{2})?/?$" {
+        add_header Cache-Control no-cache;
```

Comment:
Don't forget to mirror these changes in infrastructure/conf/nginx-conf-deploy/default

Reply:
Completely forgot about that 😞 

Added in [add no cache header to nginx deploy conf](https://github.com/GCTC-NTGC/gc-digital-talent/pull/4988/commits/a44179d016416a6845b70c576d4d8c1c9236ec40)